nodejs/device/link.sh

```diff
@@ -0,0 +1,15 @@
+# !/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for
+# full license information.
+set -e 
+
+NODE_ROOT=$1
```

Comment:
`NODE_ROOT` is the root of the Node SDK repo, right? I think something like `NODE_SDK_ROOT` would be more clear.